.github/workflows/ubuntu.yml

```diff
@@ -113,12 +120,13 @@ jobs:
           - elixir-version: 'master'
             otp-version: '23.1'
             os:  'ubuntu-20.04'
-          - elixir-version: 'master'
+          - elixir-version: 'main'
```

Comment:
This was actually a fix. Since OTP 25, Elixir no longer has `master`, but `main`.